sdk/cosmos/azure-cosmos/test/test_session_token_helpers.py

```diff
@@ -0,0 +1,165 @@
+# The MIT License (MIT)
+# Copyright (c) Microsoft Corporation. All rights reserved.
+
+import random
+import unittest
+
+import pytest
+
+import azure.cosmos.cosmos_client as cosmos_client
+import test_config
+from azure.cosmos import DatabaseProxy
+from azure.cosmos._change_feed.feed_range_internal import FeedRangeInternalEpk
+from azure.cosmos._routing.routing_range import Range
+
+COLLECTION = "created_collection"
+DATABASE = "created_db"
+@pytest.fixture(scope="class")
+def setup():
+    if (TestSessionTokenHelpers.masterKey == '[YOUR_KEY_HERE]' or
+            TestSessionTokenHelpers.host == '[YOUR_ENDPOINT_HERE]'):
+        raise Exception(
+            "You must specify your Azure Cosmos account values for "
+            "'masterKey' and 'host' at the top of this class to run the "
+            "tests.")
+    test_client = cosmos_client.CosmosClient(TestSessionTokenHelpers.host, test_config.TestConfig.masterKey),
+    created_db = test_client[0].get_database_client(TestSessionTokenHelpers.TEST_DATABASE_ID)
+    return {
+        DATABASE: created_db,
+        COLLECTION: created_db.get_container_client(TestSessionTokenHelpers.TEST_COLLECTION_ID)
+    }
+
+def create_split_ranges():
+    return [        # split with two children
+                   ([(("AA", "DD"), "0:1#51#3=52"), (("AA", "BB"),"1:1#55#3=52"), (("BB", "DD"),"2:1#54#3=52")],
+                    ("AA", "DD"), "1:1#55#3=52,2:1#54#3=52"),
+                    # same range different partition key range ids
+                   ([(("AA", "DD"), "1:1#51#3=52"), (("AA", "DD"),"0:1#55#3=52")],
+                    ("AA", "DD"), "0:1#55#3=52"),
+                    # split with one child
+                   ([(("AA", "DD"), "0:1#51#3=52"), (("AA", "BB"),"1:1#55#3=52")],
+                    ("AA", "DD"), "0:1#51#3=52,1:1#55#3=52"),
+                    # several ranges being equal to one range
+                   ([(("AA", "DD"), "0:1#42#3=52"), (("AA", "BB"), "1:1#51#3=52"),
```

Comment:
Good to call out the GLSN distinction in the "several ranges being equal to one range" scenario.

Reply:
Added the distinction